rbac/management/role/definer.py

```diff
@@ -131,16 +140,16 @@ def seed_permissions(tenant):
                                         f"Created permission {permission.permission} "
                                         f"for tenant {tenant.schema_name}."
                                     )
-                                    # current_permission_ids.add(permission.id)
+                                current_permission_ids.add(permission.id)
                         except Exception as e:
                             logger.error(
                                 f"Failed to update or create permissions for: "
                                 f"{app_name}:{resource} for tenant: {tenant.schema_name} with error: {e}"
                             )
-
-            # TODO: Remove permissions that are no longer exist, could enable later after enforcing
-            # all available permission from the permissions files (there might be some custom ones for
-            # Costmanagement or Remediations currently)
-            # Permission.objects.exclude(id__in=current_permission_ids).delete()
-            # Override delete methods for Permission to remove the related Access objects
+        perms_to_delete = Permission.objects.exclude(id__in=current_permission_ids)
+        logger.info(
+            f"[READ-ONLY] Deleting the following '{perms_to_delete.count()}' permission(s): {perms_to_delete.values()}"
```

Comment:
Same here as above, not sure if [READ ONLY] fully indicates that the operation isn't actually being performed at the time.